components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -1092,6 +1058,8 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
     }
 
     async tsAssignSlot(ctx: TraceContext, teamSubscriptionId: string, teamSubscriptionSlotId: string, identityStr: string | undefined): Promise<void> {
+        TraceContext.redactJsonRPCParameters(ctx, [2]);
```

Comment:
See [comment above](https://github.com/gitpod-io/gitpod/pull/7032#discussion_r761889048), especially this part:

> Having a single line at the beginning of each method is also fine with me. Would safe us from the strange "indentify by index" problem (cmp. redactJsonRPCParameters). AND would give us "named parameters", so rpc.jsonrpc.parameters.workspaceId instead of rpc.jsonrpc.parameters.0. Especially the last two make me reconsider... :thinking: